src/popup/ui.js

```diff
@@ -63,3 +78,73 @@ function createButtonsDiv(node, onReloadItems) {
 	buttons.appendChild(archiveButton)
 	return buttons
 }
+
+function loadLabelSelection(article, labels){
+	document.getElementById('content').style = 'display: none;'
+
+	var labelPage = document.getElementById('labelPage')
+	labelPage.style = 'display: flex;'
+
+	var ckBoxContainer = document.getElementById('labelList')
+
+	labels.forEach((item) => {	
+		var li = document.createElement('li');
+
+		var checkbox = document.createElement('input');
+		checkbox.name = "checkboxLabel";
+		checkbox.type = "checkbox";
+		checkbox.value = item.id;
+		checkbox.id = item.id;
+
+		var label = document.createElement('label')
+		label.htmlFor = item.id;
+		label.appendChild(document.createTextNode(' ' + item.name));
+
+		if(article.labels !== null) {
+			article.labels.forEach((gg) => {	
+				if(gg.id == item.id)
+					checkbox.checked = true;
+			})		
+		}
```

Comment:
This can be a bit simplified with modern JS features!

```suggestion
		const isChecked = !!article.labels?.find((label) => label.id === item.id)
		checkbox.checked = isChecked
```